src/coreclr/vm/exceptionhandling.cpp

```diff
@@ -2300,6 +2306,46 @@ bool ExceptionTracker::HandleNestedExceptionEscape(StackFrame sf, bool fIsFirstP
     return fResult;
 }
 
+#if defined(DEBUGGING_SUPPORTED)
+BOOL NotifyDebuggerOfStub(Thread* pThread, Frame* pCurrentFrame)
+{
+    LIMITED_METHOD_CONTRACT;
+
+    BOOL fDeliveredFirstChanceNotification = FALSE;
+
+    // <TODO>
+    // Remove this once SIS is fully enabled.
+    // </TODO>
+    extern bool g_EnableSIS;
```

Comment:
Delete this from here and everywhere else? It seems to be unconditionally enabled, so this TODO is no longer relevant.

Reply:
Deleted